show-me-the-visualization/README.md

```diff
@@ -119,14 +119,14 @@ index = 4 x = -1.02 y = -0.28 slope = 14.40 delta_x = -0.02
 index = 5 x = -1.00 y = -0.00 slope = 14.01 delta_x = -0.00
 ```
 
-The last iteration has `-1.00` as x, which is what we expected.
+The last iteration has `x = -1.00`, which is what we expected.
 
 ## Bindings
 
-Emscripten can handle simple types like float and int, but needs help exposing more complex types to JavaScript like the iterations property.
-We need to use [value_object](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#value-types) to expose the Iteration struct and [register_vector](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#built-in-type-conversions) as the iterations property type.
+Emscripten can handle simple types like `float` and `int`, but needs help exposing more complex types to JavaScript like the `iterations` property.
+We need to use [`value_object`](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#value-types) to expose the `Iteration` `struct` and [`register_vector`](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#built-in-type-conversions) as the `iterations` property type.
```

Comment:
Links are already rendered differently, they do not need more highlighting
```suggestion
We need to use [value_object](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#value-types) to expose the `Iteration` `struct` and [register_vector](https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#built-in-type-conversions) as the `iterations` property type.
```

Reply:
strictly speaking I don't care to highlight them, but I want to emphasize that I'm referring to something in the code. It's your call though.